src/actions/index.js

```diff
@@ -30,3 +30,17 @@ export const fetchSyntaxes = dispatch => (
     .then(response => response.json())
     .then(json => dispatch(setSyntaxes(json)))
 );
+
+export const postSnippet = snippet => dispatch => (
+  fetch('http://api.xsnippet.org/snippets', {
+    method: 'POST',
+    mode: 'cors',
```

Comment:
If I'm not mistaken, `cors` is default value. I'd suggest to do not mention this, as we don't set it explicitly for other requests we make.

Reply:
will remove